tests/tools/commands/test_venv.py

```diff
@@ -798,3 +798,45 @@ def create_venv_pex():
     assert os.path.exists(venv_dir)
     assert not os.path.exists(venv_pex)
     assert not os.path.exists(pex_root)
+
+@pytest.mark.parametrize(
+    "system_site_package", [pytest.param(flag, id=str(flag)) for flag in [True, False]]
+)
+def test_system_site_package(
+    tmpdir,
+    system_site_package,  # type: Layout.Value
+):
+    # type: (...) -> None
+    pex_root = os.path.join(str(tmpdir), "pex_root")
+
+    def create_venv_pex():
+        # type: () -> str
+        venv_pex = os.path.join(str(tmpdir), "venv.pex")
+        args=[
+            "--pex-root",
+            pex_root,
+            "--runtime-pex-root",
+            pex_root,
+            "-o",
+            venv_pex,
+            "--include-tools",
+        ]
+        if system_site_package.value:
+            args.append("--system-site-packages")
+        run_pex_command(args=args).assert_success()
+        return venv_pex
+
+    venv_dir = os.path.join(str(tmpdir), "venv_dir")
+    assert not os.path.exists(venv_dir)
+
+    venv_pex = create_venv_pex()
+    subprocess.check_call(args=[venv_pex, "venv", venv_dir], env=make_env(PEX_TOOLS=True))
+    assert os.path.exists(venv_dir)
+    assert os.path.exists(venv_pex)
+    assert os.path.exists(pex_root)
+
+    pyvenv_content = open(os.path.join(venv_dir, "pyvenv.cfg")).read()
+    print(pyvenv_content)
```

Comment:
These are not great tests :)

How about something like this:
```python
venv = Virtualenv.enclosing(os.path.join(venv_dir, "bin", "python"))
venv_interpreter = venv.interpreter
venv_base_interpreter = venv_interpreter.resolve_base_interpreter()
assert venv_interpreter != venv_base_interpreter, (
    "The venv base interpreter should be the system interpreter; not the venv interpreter itself."
)

venv_sys_path = venv.sys_path
venv_interpreter_site_packages = venv_interpreter.site_packages
system_site_packages = venv_base_interpreter.site_packages
assert not set(venv_interpreter_site_packages).intersection(system_site_packages)
assert system_site_packages in venv_sys_path
```

You may need to experiment a bit to get the set math right, but this sort of thing gives you a real test that will succeed or fail depending on whether system site packages are actually included. Currently the test requires a human to go look at the output you print, which is not great. The test can break and no-one may know because they don't read the print output.

Reply:
Sorry currently it's WIP. I forgot to mark it as draft. Thanks for your code snippet :)

Reply:
Now it's ready for review!